<launch>
    <!-- ros2 bag play /home/borges/humble_ws/src/ratslam/bags/irat_aus_28112011/irat_aus_28112011.db3 __clock -->
    <node
        pkg="ratslam"
        exec="ratslam_lv"
        name="RatSLAMLocalViewCells"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="config_file" value="$(find-pkg-share ratslam)/config/config_surveyor_t2.txt.in" />
        <param name="use_sim_time" value="true" />
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
    </node>

    <node
        pkg="ratslam"
        exec="ratslam_pc"
        name="RatSLAMPoseCells"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="config_file" value="$(find-pkg-share ratslam)/config/config_surveyor_t2.txt.in" />
        <param name="use_sim_time" value="true" />
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
    </node>

    <node
        pkg="ratslam"
        exec="ratslam_em"
        name="RatSLAMExperienceMap"
        output="screen"
        args="--ros-args --remap _image_transport:=compressed">
        <param name="config_file" value="$(find-pkg-share ratslam)/config/config_surveyor_t2.txt.in" />
        <param name="use_sim_time" value="true" />
        <param name="media_path" value="$(find-pkg-share ratslam)/media" />
        <param name="image_file" value="irat_sm.tga" />
    </node>      
            
<!--  
    <arg name="config_file" default="$(find-pkg-share ratslam)/config/config_irataus.yaml"/>
    <node
        pkg="ratslam"
        exec="ratslam_lv"
        name="RatSLAMLocalViewCells"
		namespace="irat_red"
        output="screen">
    </node> -->
</launch>

